collectors/salesforce/collector.js

```diff
@@ -14,7 +14,7 @@ const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
 const parse = require('@alertlogic/al-collector-js').Parse;
 const utils = require("./utils");
 var jwt = require('jsonwebtoken');
-var request = require('request');
+const RestServiceClient = require('@alertlogic/al-collector-js/al_util').RestServiceClient;
```

Comment:
In the latest version or `al-collector-js` this class is exported via `index.js`.
Could you please change it to `require('@alertlogic/al-collector-js').RestServiceClient`
Also please update deps in `package.json`